l2-contracts/contracts/bridge/L2SharedBridge.sol

```diff
@@ -114,15 +119,27 @@ contract L2SharedBridge is IL2SharedBridge, IL2SharedBridgeLegacy, Initializable
     /// @param _assetData The data that is passed to the asset contract
     function withdraw(bytes32 _assetInfo, bytes calldata _assetData) external override {
```

Comment:
Shall we test it here or just integration is fine?

Reply:
Test what?